packages/domain/src/use-cases/transactions/BroadcastingError.ts

```diff
@@ -18,7 +27,7 @@ export class BroadcastingError extends Error {
   /**
    * @internal
    */
-  constructor(readonly reason: BroadcastingErrorReason) {
+  constructor(readonly reason: BroadcastingErrorReason, readonly fallback?: RequestFallback) {
```

Comment:
`fallback` is optional for now that not all codepaths that generates it are changed to produce one. After all changes I'll review if it's possible to make it non-optional.